docs/source/howto/query.rst

```diff
@@ -150,7 +150,7 @@ In case you want all calculation jobs with state ``finished`` or ``excepted``, y
         },
     )
 
-You can negate a filter by adding an exclamation mark in front of the operator.
+You can **negate** a filter by adding an exclamation mark, ``!`` or a ``~``, in front of the operator, that is a ``NOT`` operatror.
```

Comment:
Think the original phrasing could be mistaken to mean that the filter itself is no longer considered. Maybe the following is more correct?
```suggestion
A filter can be turned into its associated **negation** by adding an exclamation mark, ``!``, in front of the operator.
```
I also wonder whether we should really be providing two different values even if both work. Are both identical? If so, I would just provide one, to keep things simple. If they behave differently, we should document here how.